lib/core/src/lib/viewer/components/viewer.component.spec.ts

```diff
@@ -525,6 +525,33 @@ describe('ViewerComponent', () => {
             });
         });
 
+        describe('Close Button', () => {
+
+            it('should show close button on left side when closeButtonPosition is left and allowGoBack is true', () => {
+                component.allowGoBack = true;
+                component.closeButtonPosition = 'left';
+                fixture.detectChanges();
+
+                expect(element.querySelector('.adf-left-close-button')).not.toBeNull();
+            });
+
+            it('should show close button on right side when closeButtonPosition is right and allowGoBack is true', () => {
```

Comment:
Could you additionally check if in that case close button is not displayed at left side?